buildpacks/ruby/src/main.rs

```diff
@@ -126,7 +126,9 @@ impl Buildpack for RubyBuildpack {
             .map_err(|error| RubyBuildpackError::MissingGemfileLock(lockfile, error))?;
         let gemfile_lock = GemfileLock::from_str(&lockfile_contents).expect("Infallible");
         let bundler_version = gemfile_lock.resolve_bundler("2.5.23");
+        tracing::info!(version_bundler = bundler_version.to_string());
```

Comment:
As this is a macro, it doesn't need to obey Rust naming rules which are slightly different than OTel naming standards. 

```suggestion
        tracing::info!(cnb.ruby.bundler.version = bundler_version.to_string());
```